# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: netwerk/wifi/nsWifiMonitor.h
# Commit: 6cbe2549217c
# Full Hash: 6cbe2549217c7cc45f19dce0e064e8ad94601969
# Author: David Parks <daparks@mozilla.com>
# Date: 2023-07-21 04:09:26
# Description:
#   Bug 1842860: Increase stack size in WifiMonitor on MacOS r=haik,necko-reviewers,kershaw
#   
#   We have been seeing the stack issue from bug 1776210, but on MacOS 10.12 Sierra.  We are increasing stack size for all versions.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D183611
# ==============================================================================

diff -r b0f40ba976ad -r 6cbe2549217c netwerk/wifi/nsWifiMonitor.h
--- a/netwerk/wifi/nsWifiMonitor.h	Fri Jul 21 00:54:10 2023 +0300
+++ b/netwerk/wifi/nsWifiMonitor.h	Thu Jul 20 22:11:27 2023 +0000
@@ -33,9 +33,9 @@
 #define WIFI_SCAN_INTERVAL_MS_PREF "network.wifi.scanning_period"
 
 #ifdef XP_MACOSX
-// Use a larger stack size for the monitor thread on macOS 13+
-// to accommodate Core WLAN making large stack allocations.
-#  define kMacOS13MonitorStackSize (512 * 1024)
+// Use a larger stack size for the wifi monitor thread of macOS, to
+// accommodate Core WLAN making large stack allocations.
+#  define kMacOSWifiMonitorStackSize (512 * 1024)
 #endif
 
 struct WifiListenerHolder {
